src/parser.rs

```diff
@@ -88,17 +88,11 @@ pub fn parse_command(lexer: &mut Lexer, script: &mut Script) -> Result<bool, (Pa
 				let character = CharacterName(inline(lexer.string())?);
 				let state = StateName(inline(lexer.string())?);
 				let position = position(lexer)?;
-				let mut is_end = false;
 				let instance_name = match inline(lexer.peek())? {
-					None | Some(Token::Terminator) => {
-						is_end = true;
-						None
-					}
-					Some(Token::Identifier(identifier)) if identifier == "with" => None,
-					Some(Token::String(_)) => Some(InstanceName(inline(lexer.string())?)),
-					Some(_) => return Err((ParserError::UnexpectedToken, Token::Terminator)),
+					Some(Token::String(_)) => Some(InstanceName(lexer.string().unwrap())),
+					_ => None,
 				};
```

Comment:
Probably best to add a new line after this match expression.

Reply:
Done